tests/vmi_multus_test.go

```diff
@@ -698,6 +698,32 @@ var _ = Describe("[Serial]SRIOV", func() {
 			}
 		}
 
+		It("should create virt-launcher pod with CAP_SYS_RESOUCE capability", func() {
+			tests.EnableFeatureGate("SRIOVLiveMigration")
+
+			vmi := getSriovVmi([]string{sriovnet1}, defaultCloudInitNetworkData())
+			vmi = startVmi(vmi)
+			vmi = waitVmi(vmi)
+
+			By("Looking up for VMI virt-launcher pod using VMI's label")
+			pods, err := virtClient.CoreV1().Pods(tests.NamespaceTestDefault).List(tests.UnfinishedVMIPodSelector(vmi))
+			Expect(err).ToNot(HaveOccurred())
+			Expect(pods.Items).NotTo(BeEmpty())
+			pod := pods.Items[0]
+
+			foundContainer := false
+			for _, container := range pod.Spec.Containers {
+				if container.Name == "compute" {
+					foundContainer = true
+					Expect(container.SecurityContext.Capabilities.Add).
+						To((ContainElement(k8sv1.Capability("SYS_RESOURCE"))))
+					break
+				}
+			}
```

Comment:
How about:
```
computeContainers := filterContainersByName("compute")
Expect(computeContainers).NotTo(BeEmpty(), ...)
for c := range computeContainers {...}
```

Reply:
I found a function in tests package that does exactly that, I will use it

Reply:
> For the last commit: I think you should add to `Live Migration for VM's with SRIOV interfaces should be blocked.` something like "even though the feature-gate is open".

Done